Tutorials/WinForms/Classic/create-persistent-classes-and-connect-xpo-to-database.md

```diff
@@ -0,0 +1,99 @@
+    ```csharp
+    public string ContactName {
+        get { return string.Concat(FirstName, " ", LastName); }
+    }
+    ```
+* Add the [PersistentAliasAttribute](https://docs.devexpress.com/XPO/DevExpress.Xpo.PersistentAliasAttribute) to the `ContactName` property. With this attribute, XPO will be able to include this property in a filter. The expression passed to the attribute constructor should follow the [Criteria Language Syntax](https://docs.devexpress.com/CoreLibraries/4928/devexpress-data-library/criteria-language-syntax). XPO uses the Persistent Alias expression to build a SQL command compatible with the target database.
+    ```csharp
+    [PersistentAlias("concat(FirstName, ' ', LastName)")]
+    public string ContactName {
+        get { return string.Concat(FirstName, " ", LastName); }
+    }
+    ```
+* Use the same approach to add the `Order` class with the `ProductName`(String), `OrderDate`(DateTime), and `Freight`(decimal) properties. Check our example code in case of any difficulty:  
+  [Order.cs](/Tutorials/WinForms/Classic/CS/DataAccess/Order.cs)  
+  [Order.vb](/Tutorials/WinForms/Classic/VB/DataAccess/Order.vb)
+* Add the `Orders` and `Customer` properties to create a [relationship](https://docs.devexpress.com/XPO/2041/concepts/relationships-between-objects) between the **Customer** and **Order** classes.
+    ```csharp
+    // Customer.cs
+    [Association("CustomerOrders")]
+    public XPCollection<Order> Orders {
+        get { return GetCollection<Order>(nameof(Orders)); }
+    }
+    
+    // Order.cs
+    private Customer fCustomer;
+    [Association("CustomerOrders")]
+    public Customer Customer {
+        get { return fCustomer; }
+        set { SetPropertyValue(nameof(Customer), ref fCustomer, value); }
+    }
+    ```
+* Add a reference to the **System.Configuration** assembly. This assembly contains classes required to read the connection string settings from the application configuration file [Connection Strings and Configuration Files](https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/connection-strings-and-configuration-files).
+* Add the `ConnectionHelper` class and copy the code from the [ConnectionHelper.cs](/Tutorials/WinForms/Classic/CS/DataAccess/ConnectionHelper.cs) ([ConnectionHelper.vb](/Tutorials/WinForms/Classic/VB/DataAccess/ConnectionHelper.vb)) file. In most cases, this generic helper can be used without modifications. The complete documentation about possible connection settings is provided in the XPO documentation: [Connecting to a Data Store](https://docs.devexpress.com/XPO/2020/feature-center/connecting-to-a-data-store).
+* Open the *Program.cs* file and add the `ConnectionHelper.Connect` method call to the `Main` method (the *Program.vb* file does not exist in a VB.NET project by defult, use the `Form1` class constructor, instead):
```

Comment:
Firstly, ' by defult' is incorrect, and secondly, do not use 'by default'.

Reply:
a7c4070d23fb271cea65a538d6dac3c2a81685e0